### PR TITLE
Simple renames

### PR DESCRIPTION
**Pull request by dougbu**
- "Form Data" -> "form URL-encoded data"
  - #145
- rename Salesforce receiver and configuration in Core
  - #193
  - also lowercase Pusher reciever name in Core for consistency

nits: accept VS suggestions in changed files e.g. more `var`

### CODE REVIEW

File: src/Microsoft.AspNet.WebHooks.Receivers/WebHooks/WebHookReceiver.cs [hunk 1]
```diff
@@ -13,16 +13,14 @@
 using System.Threading.Tasks;
 using System.Web.Http;
 using System.Web.Http.Controllers;
-using System.Web.Http.Dependencies;
```
Comment (dougbu) [inline src/Microsoft.AspNet.WebHooks.Receivers/WebHooks/WebHookReceiver.cs]: If a changed line in the ASP.NET code doesn't mention "form data", VS changed it when I accepted its suggestions or when I cleaned up `using`s afterward.

File: src/Microsoft.AspNet.WebHooks.Receivers/WebHooks/WebHookReceiver.cs [hunk 2]
```diff
@@ -55,37 +53,37 @@ protected WebHookReceiver()
...
             where T : JToken
         {
-            HttpConfiguration config = request.GetConfiguration();
+            var config = request.GetConfiguration();
 
             // Check that there is a request body
             if (request.Content == null)
             {
-                string msg = ReceiverResources.Receiver_NoBody;
+                var msg = ReceiverResources.Receiver_NoBody;
```
Comment (dougbu) [inline src/Microsoft.AspNet.WebHooks.Receivers/WebHooks/WebHookReceiver.cs]: LMK if you want "msg" -> "message" and perhaps "config" -> "configuration" in this PR. If so, I'll send out a separate PR to make those changes globally.

File: src/Microsoft.AspNetCore.WebHooks.Receivers.Salesforce/SalesforceWebHookAttribute.cs [hunk 3]
```diff
@@ -17,12 +17,7 @@ namespace Microsoft.AspNetCore.WebHooks
     /// <see cref="System.Xml.Linq.XElement"/> or <see cref="SalesforceNotifications"/>.
     /// </para>
     /// <para>
-    /// The '<c>MS_WebHookReceiverSecret_SalesforceSoap</c>' configuration value contains Salesforce Organization IDs.
-    /// The Organizational IDs can be found at <see href="http://www.salesforce.com"/> under
-    /// <c>Setup | Company Profile | Company Information</c>.
-    /// </para>
```
Comment (dougbu) [inline src/Microsoft.AspNetCore.WebHooks.Receivers.Salesforce/SalesforceWebHookAttribute.cs]: In another branch (not a PR yet), I correct this information and move it to the `MvcBuilder` / `MvcCoreBuilder` extension methods.

File: src/Microsoft.AspNet.WebHooks.Receivers/WebHooks/WebHookReceiver.cs (hunk 2 above)
Comment (rynowak) [reply src/Microsoft.AspNet.WebHooks.Receivers/WebHooks/WebHookReceiver.cs]: Separate is fine

**Comment by dougbu** [comment]
4938597d17 for #145 and ff7ed820e7 for #193

**Pull request closed without merging**